### PR TITLE
chore(Popper):remove popperMatchesTriggerWidth prop

### PR DESCRIPTION
**Pull request by Dominik-Petrik**
closes: #286

### CODE REVIEW

File: packages/eslint-plugin-pf-codemods/lib/rules/v5/popper-remove-popperMatchesTriggerWidth.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { renameProp } = require('../../helpers');
+
+  // https://github.com/patternfly/patternfly-react/pull/https://github.com/patternfly/patternfly-react/pull/8724
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/lib/rules/v5/popper-remove-popperMatchesTriggerWidth.js:3]: Just need to clean this comment up

File: packages/eslint-plugin-pf-codemods/lib/rules/v5/popper-remove-popperMatchesTriggerWidth.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  module.exports = {
+    meta: { fixable: 'code' },
+    create: 
+    renameProp(
+      'Popper',
+      {'popperMatchesTriggerWidth': ''},
+      node =>  `popperMatchesTriggerWidth prop has been removed for ${node.name.name}, minWidth now covers this by default. Minimum and maximum width can be modified via the new minWidth and maxWidth properties`
```
Comment (thatblindgeye) [inline packages/eslint-plugin-pf-codemods/lib/rules/v5/popper-remove-popperMatchesTriggerWidth.js:10]: Slight tweak, let me know what you think:

```suggestion
      node =>  `popperMatchesTriggerWidth prop has been removed for ${node.name.name}.  The width can instead be modified via the new minWidth, maxWidth, and width properties`
```

Or something more similar to the description from the README.

File: packages/pf-codemods/README.md [hunk 3]
```diff
@@ -638,7 +639,23 @@ Out:
   }}
 />
 ```
+### popper-remove-popperMatchesTriggerWidth [(#8724)](https://github.com/patternfly/patternfly-react/pull/8724)
+
+We've removed the `popperMatchesTriggerWidth` prop from Popper, minWidth, maxWidth and width props now can be used to modify the popper width.
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:644]:
```suggestion
We've removed the `popperMatchesTriggerWidth` prop from Popper. `minWidth`, `maxWidth`, and `width` props can instead be used to modify the Popper width.
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: packages/pf-codemods/README.md (hunk 3 above)
Comment (Dominik-Petrik) [reply packages/pf-codemods/README.md]: Thanks for the feedback, I've made the changes.

**Review by thatblindgeye — APPROVED** [review]

File: packages/pf-codemods/README.md (hunk 3 above)
Comment (thatblindgeye) [reply packages/pf-codemods/README.md]: Thank you! And awesome job on this 🔥

**Review by kmcfaul — APPROVED** [review]
LGTM. 

I wonder if we want to add a note/warning for the components that got `minWidth` added but since that's not breaking (as most of those components only used the removed prop internally and didn't expose it) the release notes should cover that.

**Review by gitdallas — APPROVED** [review]

**Pull request merged**